SUMO-254673 doc changes for v 1 to v 2 migrations apps (#5019)
SumoLogic/sumologic-documentation: docs/integrations/app-development/bitbucket.md

### Step 4: Enable Bitbucket Event-Key tagging at Sumo Logic

> Sumo Logic needs to understand the event type for incoming events (for example, repo:push events). To enable this, the [X-Event-Key](https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html#EventPayloads-HTTPheaders) event type is automatically added to the [Fields](/docs/manage/fields) during installation of the app.

Reviewer: ```suggestion
To properly identify the event type for incoming events (for example, repo:push events), Sumo Logic automatically adds the [X-Event-Key](https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html#EventPayloads-HTTPheaders) event type to the [Fields](/docs/manage/fields) during app installation.
```

> 

Reviewer: remove extra lines/spaces.
